docking/pose_dock.py

```diff
@@ -80,9 +89,20 @@ def pose_dock(
             rigidWater=False,
         )
 
-        bps, masses = dock_setup.combine_potentials(
-            guest_ff_handlers, guest_mol, host_system
-        )
+        if skip_errors:
+            try:
+                bps, masses = dock_setup.combine_potentials(
+                    guest_ff_handlers, guest_mol, host_system
+                )
+            except Exception as err:
+                print(f"Error: There was a problem setting up {guest_name}")
```

Comment:
Curious to know what were some of the errors you've encountered?

Reply:
Usually it was when I had problems with my input guest structures -- missing hydrogens or atoms too close together.

Reply:
Ah okay - would be nice to have this be more informative but we can add more diagnostic information later.